aip/client-libraries/4233.md

```diff
@@ -27,7 +27,7 @@ are met:
 - The request message contains an `int32 page_size` field.
 - The request message contains a `string page_token` field.
 - The response message contains a `string next_page_token` field.
-- The response field contains one non-primitive `repeated` field.
+- The response message contains one non-primitive `repeated` field.
```

Comment:
```suggestion
- The response message contains at least one non-primitive `repeated` field.
```

I felt this would provide additional clarity, but wanted to get your thoughts on this before making that change.

Reply:
I prefer the original wording. Per [AIP-132 Response Message](https://aip.dev/132#response-message), the response message should only have one `repeated` field. There are some circumstances in which _specific_ extra repeated fields are permitted, but those are covered by their respective AIPs. The newly suggested wording opens it up for any number of repeated fields being allowed and that is counter to what we want.